docs/source/notebooks/elements.ipynb

```diff
@@ -107,6 +107,7 @@
     "                                  connection=connection)\n",
     "globals().update(**objects)\n",
     "\n",
+    "data['models']['simulation']['npoint'] = 1000000\n",
```

Comment:
This change appears from #88. Once that PR is merged, let's rebase this PR branch so that we only have changes specific to this PR.

Reply:
Rebased: https://github.com/NSLS-II/sirepo-bluesky/pull/89#event-7566620792.